torch/csrc/jit/passes/mkldnn_rewrite.h

```diff
@@ -18,15 +18,18 @@ namespace jit {
 
 namespace mkldnn {
 
+using AttrFunction = std::function<ideep::attr_t(
+    std::vector<c10::optional<at::Scalar>>,
+    c10::optional<std::string>)>;
+
 struct PostOp {
-  ideep::attr_t op_attr;
+  AttrFunction attr_function;
+  std::vector<std::string> op_input_list;
+  std::string op_list_construct;
```

Comment:
This field comes from `op_input_list`. Hence, I think we can exclude this field from this structure and construct it at runtime.

Reply:
Removed `op_list_construct` from the struct.